churn-risk/src/churn_predictor.py

```diff
@@ -24,45 +24,38 @@ def __init__(self):
         self.predicted_df = self.model.predict(self.h2o_test_df).as_data_frame()
         self.contributions_df = self.model.predict_contributions(self.h2o_test_df).drop('BiasTerm').as_data_frame()
 
-    def get_churn_rate(self, row_index: int):
+    def get_churn_rate(self, row_index: int) -> float:
         return round(float(self.predicted_df["TRUE"][row_index]) * 100, 2)
 
-    def get_shap(self, row_index: int):
+    def get_shap(self, row_index: int) -> list:
```

Comment:
You should use `List` from `typing` library to annotate lists. `list` is possible too but only from version 3.9 (if I remember correctly). This applies to all occurrences here in the file.

Reply:
Good to know, thanks!